apps/admin/src/app/permissions/management/permission-management-page.component.html

```
@@ -0,0 +1,265 @@
+                    </mat-select></mat-form-field>
+                    <button mat-stroked-button type="button" [disabled]="!store.canUpdate() || !!role.archivedAt" (click)="store.addGroupAssignment(groupAssignment.value); groupAssignment.value = null">
+                      <mat-icon>group_add</mat-icon>Adicionar grupo
+                    </button>
+                  </div>
+
+                  <div class="assignment-list">
+                    @for (assignment of role.assignments; track $index; let assignmentIndex = $index) {
+                      <article class="assignment">
+                        <header><div><mat-icon>{{ assignment.personId ? 'person' : 'groups' }}</mat-icon>
+                          <span><strong>{{ store.assignmentName(assignment) }}</strong>
+                            @if (assignment.personId && !store.roles().flatMap(r => r.assignments).find(a => a.personId === assignment.personId)?.subjectHasLinkedUser) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)